resources/teamcity/includes/tc-helpers.inc.sh

```diff
@@ -66,3 +66,19 @@ set_variables_for_emscripten() {
   export EMSCRIPTEN_BASE="${EMSCRIPTEN_BASE:-${HOME}/emsdk/upstream/emscripten}"
   export KEYMAN_USE_EMSDK=1
 }
+
+upload_help() {
```

Comment:
I wonder if we should use a prefix for all the tc-*.inc.sh functions:

```suggestion
tc_upload_help() {
```

This would help avoid conflicts in the future